### ISSUE TITLE
Keyboard focus visible on disabled contextual menu item violates Microsoft's Focus Order (MAS 2.4.3)

### ISSUE THREAD

**Issue by vparames86** [microsoft/fluentui#20711]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^8.41.2"
- **Browser and OS versions**: Microsoft Edge, Windows 10

### Describe the issue:
There was an issue which already got created (https://github.com/microsoft/fluentui/issues/9397) and closed, but this behavior is not following Microsoft's Guideline.
**MAS Reference**:
MAS 2.4.3 - Focus Order
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d
**WCAG Reference**:
https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

#### Actual behavior:

Keyboard focus visible on disabled contextual menu item. 

#### Expected behavior:

Keyboard focus shouldn't be visible on disabled contextual menu item. Please enable (inactive api) in https://docs.microsoft.com/en-us/javascript/api/react-internal/icontextualmenuitem?view=office-ui-fabric-react-latest which is deprecated now.

**Comment by gouttierre** [comment]
Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@layershifter - Would you be able to confirm if this is a regression, or if this behavior is an issue? The previous issue (#9397)  was closed as by design. [See comment ](https://github.com/microsoft/fluentui/issues/9397#issuecomment-500926867) 
cc @smhigley

**Comment by cschlechty** [comment]
This is a misinterpretation of the rule. Disabled elements are allowed to be focusable. The rule does not require that disabled elements cannot receive focus. Happy to discuss off-line (There are other issues with the filing of this bug)
